<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticDialog</class>
 <widget class="QDialog" name="StatisticDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>任务统计报表</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QRadioButton" name="radioBtnToday">
       <property name="text">
        <string>今日报表</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioBtnWeek">
       <property name="text">
        <string>本周报表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGenerate">
       <property name="text">
        <string>重新生成报表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExportPng">
       <property name="text">
        <string>导出为PNG</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QChartView" name="chartViewPie">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="renderHint">
        <enum>QPainter::Antialiasing</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QChartView" name="chartViewLine">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="renderHint">
        <enum>QPainter::Antialiasing</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelInfo">
     <property name="text">
      <string>报表信息：完成率、分类占比、逾期率统计</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <!-- 删除原有错误的connections，改为代码中绑定 -->
 <connections/>
</ui>
